### PR TITLE
Contents migration

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

**Review by jhlee-young — APPROVED** [review]

File: content/en/docs/learn/brand-assets.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Brand Assets
+weight: 20
+---
+
+[XPLA Chain brand asset](/XPLA_Chain_brand_asset.zip)
```
Comment (psy2848048) [inline content/en/docs/learn/brand-assets.md:6]: Will the link work after production deployment?

Comment (caramis) [reply content/en/docs/learn/brand-assets.md:6]: I missed it. Thank you for the good review!

Comment (caramis) [reply content/en/docs/learn/brand-assets.md]: Fixed.

File: content/en/docs/learn/xpla-vault/send.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Send
+weight: 50
+---
+
+You can send tokens to other XPLA Vaults using the **Send** feature of XPLA Vault. If this is your first time using XPLA Vault, follow the [XPLA Vault tutorial]({{< ref "xpla-vault" >}}).
```
Comment (psy2848048) [inline content/en/docs/learn/xpla-vault/send.md:6]: Would be better to change the first subpage of this section

File: content/en/docs/learn/xpla-vault/manage-staking.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+title: Manage Staking
+weight: 70
+---
+
+Use this guide to manage your staking delegations in XPLA Vault. To learn how to stake or withdraw rewards, visit the [XPLA mobile vault tutorials]({{< ref "vaults/mobile-vault" >}}).
+
+If this is your first time using XPLA Vault, follow the [XPLA Vault tutorial]({{< ref "xpla-vault" >}}).
```
Comment (psy2848048) [inline content/en/docs/learn/xpla-vault/manage-staking.md:8]: Would be better to change the first subpage of this section

File: content/en/docs/learn/xpla-vault/testnet.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Testnet
+weight: 80
+---
+
+The testnet is the testing environment for the XPLA Chain. Use this guide to simulate transactions on the testnet.
+
+If this is your first time using XPLA Vault, follow the [XPLA Vault tutorial]({{< ref "xpla-vault" >}}).
```
Comment (psy2848048) [inline content/en/docs/learn/xpla-vault/testnet.md:8]: Would be better to change the first subpage of this section

File: content/en/docs/learn/xpla-vault/wallet.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
+---
+title: Wallet
+weight: 90
+---
+
+This guide is for advanced features of the XPLA Vault. If this is your first time using XPLA Vault, follow the [XPLA Vault tutorial]({{< ref "xpla-vault" >}}).
```
Comment (psy2848048) [inline content/en/docs/learn/xpla-vault/wallet.md:6]: Would be better to change the first subpage of this section

File: content/en/docs/learn/glossary.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+
+To learn how to install and get started using XPLA Vault, visit the [XPLA Vault tutorial]({{< ref "xpla-vault/vaults" >}}).
+
+To learn how to use the advanced features of XPLA Vault, visit the [XPLA Vault how-to guide]({{< ref "xpla-vault/wallet" >}}).
+
+## xplad
+
+The command line interface for interacting with a XPLA Chain node.
+
+For more information on xplad, see [`xplad` guides]({{< ref "xplad" >}}).
```
Comment (psy2848048) [inline content/en/docs/learn/glossary.md:180]: Would be better to change the first subpage of this section

File: content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+  count: 7
+```
+
+Excellent! Congratulations, you've created your first smart contract, and now know how to get developing with the XPLA Chain dApp Platform.
+
+## What's Next?
+
+So far you've walked through a simple example of a smart contract, that modifies a simple balance within its internal state. Although this is enough to make a simple dApp, you can power more interesting applications by **emitting messages**, which will enable you to interact with other contracts as well as the rest of the blockchain's module.
+
+Check out a couple more examples of smart contracts using XPLA Chain's smart contract [repo](https://github.com/xpladev/cosmwasm-contracts).
```
Comment (psy2848048) [inline content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md:223]: Broken link

File: content/en/docs/develop/tools/wallet-provider/signing-bytes.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+      {!connectedWallet && <p>Wallet not connected!</p>}
+
+      {connectedWallet && !connectedWallet.availableSignBytes && (
+        <p>This connection does not support signBytes()</p>
+      )}
+    </div>
+  );
+}
+```
+You can find this code used in context in [GitHub](https://github.com/xpladev/wallet-provider/blob/main/templates/create-react-app/src/components/SignBytesSample.tsx).
```
Comment (psy2848048) [inline content/en/docs/develop/tools/wallet-provider/signing-bytes.md:104]: Broken snippet path

File: content/en/docs/develop/tools/xplad/about-xplad.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+---
+title: About `xplad`
+weight: 10
+---
+
+`xplad` is the command-line interface and node daemon that connects to XPLA Chain and enables you to interact with the XPLA Chain. XPLA Chain core is the official Golang reference implementation of the XPLA Chain node software.
+
+This guide is for developers who want to install `xplad` and interact with XPLA Chain core without running a full node. If you want to run a full node or join a network, visit [Run a Full Node]({{< ref "run-a-full-node" >}}).
```
Comment (psy2848048) [inline content/en/docs/develop/tools/xplad/about-xplad.md:8]: Another `_index.md` issue
And the content seems to move into another md file contents

File: content/en/docs/develop/tools/xplad/install-xplad.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
+---
+title: Install `xplad`
+weight: 20
+---
+
+`xplad` is the command-line interface and daemon that connects to XPLA Chain and enables you to interact with the XPLA Chain. XPLA Chain core is the official Golang reference implementation of the XPLA Chain node software.
+
+This guide is for developers who want to install `xplad` and interact with XPLA Chain core without running a full node. If you want to run a full node or join a network, visit [Run a Full Node]({{< ref "run-a-full-node" >}}).
```
Comment (psy2848048) [inline content/en/docs/develop/tools/xplad/install-xplad.md:8]: Another _index.md issue

File: content/en/docs/develop/tools/xplad/xplad-mac.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
+---
+title: "'xplad' Mac"
+weight: 60
+---
+
+## Install `xplad` for Mac (Intel or M1)
+
+`xplad` is the command-line interface and daemon that connects to XPLA Chain and enables you to interact with the XPLA Chain. XPLA Chain core is the official Golang reference implementation of the XPLA Chain node software.
+
+This guide is for developers who want to install `xplad` and interact with XPLA Chain core without running a full node. If you want to run a full node or join a network, visit [Run a Full Node]({{< ref "run-a-full-node" >}}).
```
Comment (psy2848048) [inline content/en/docs/develop/tools/xplad/xplad-mac.md:10]: Another `_index.md` issue

**Comment by caramis** [comment]
@psy2848048 Fixed !

File: content/en/docs/full-node/manage-a-validator/register-a-validator.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+weight: 50
+---
+
+This is a detailed step-by-step guide for setting up a XPLA Chain validator. Please be aware that while it is easy to set up a rudimentary validating node, running a production-quality validator node with a robust architecture and security features requires an extensive setup.
+
+For more information on setting up a validator, see [additional resources]({{< ref "manage-a-validator#additional-resources" >}}).
+
+## Prerequisites
+
+- You have completed [how to run a full node]({{< ref "run-a-full-node" >}}), which outlines how to install, connect, and configure a node.
```
Comment (psy2848048) [inline content/en/docs/full-node/manage-a-validator/register-a-validator.md:12]: `_index.md` issue

File: content/en/docs/full-node/manage-a-validator/register-a-validator.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+- You are familiar with [xplad]({{< ref "/docs/develop/tools/xplad" >}}).
```
Comment (psy2848048) [inline content/en/docs/full-node/manage-a-validator/register-a-validator.md:13]: _index.md issue

File: content/en/docs/full-node/manage-a-validator/register-a-validator.md [hunk 14]
```diff
@@ -0,0 +1,73 @@
+---
+title: Register a Validator
+weight: 50
+---
+
+This is a detailed step-by-step guide for setting up a XPLA Chain validator. Please be aware that while it is easy to set up a rudimentary validating node, running a production-quality validator node with a robust architecture and security features requires an extensive setup.
+
+For more information on setting up a validator, see [additional resources]({{< ref "manage-a-validator#additional-resources" >}}).
```
Comment (psy2848048) [inline content/en/docs/full-node/manage-a-validator/register-a-validator.md:8]: _index.md issue

File: content/en/docs/full-node/resources.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
+---
+title: Resources
+weight: 30
+---
+
+Use these resources to set up your full node.
+
+## Text Tutorials
+
+- [Full node tutorial]({{< ref "run-a-full-node" >}})
```
Comment (psy2848048) [inline content/en/docs/full-node/resources.md:10]: _index.md issue

**Comment by caramis** [comment]
@psy2848048 Fixed !

**Review by psy2848048 — APPROVED** [review]

**Pull request merged**